test/fixtures/full_config.yml

```diff
@@ -2731,7 +2731,7 @@ Style/ClassMethodsDefinitions:
 Style/ClassVars:
   Description: Avoid the use of class variables.
   StyleGuide: "#no-class-vars"
-  Enabled: false
+  Enabled: true
```

Comment:
I have also always kinda disliked this rule although I will admit it is not common for them to be the right call

Reply:
I'll keep this in given the thread-safety and easy to fix alternative.